web/src/cypress/support/alert.ts

```diff
@@ -1,11 +1,14 @@
 import { Chance } from 'chance'
 import { DateTime } from 'luxon'
+import { sha512 } from 'js-sha512'
```

Comment:
I think we can just put generic text in there, it's generated as a hash, but just compared as a string. Something like `1:manual:createManyAlerts_#` where `#` is just a counter.

Then we don't need to worry about the overhead of calculating the hash, or the extra dep

Reply:
Interesting, I was messing around with a lot of strings and kept getting back responses that the value was invalid when inserting to the DB. I'll try this